coniferest/onnx/shape_calculators/aadforest.py

```diff
@@ -0,0 +1,13 @@
+from onnxconverter_common.registration import register_shape_calculator
+from onnxconverter_common.data_types import FloatTensorType
+from onnxconverter_common.utils import check_input_and_output_types, check_input_and_output_numbers
+
+
+def calculate_aadforest_output_shapes(operator):
```

Comment:
I don't understand what is happening here. Probably it is in onnx docs somewhere, but if not, could you please briefly describe it?